config/samples/backends/lvm/iscsi/backend.yaml

```diff
@@ -14,6 +18,9 @@ spec:
     template:
       cinderVolumes:
         lvm-iscsi:
+          nodeSelector:
```

Comment:
Similar to my comment above, maybe this is a good place to add a YAML comment that explains what to do depending on whether you have a single or 3-node OCP.